site/content/integrate/apps/structure/manifest/_index.md

```diff
@@ -1,54 +1,61 @@
+| `bindings`                                                   | [Call]({{<ref "call">}})    | The call invoked to retrieve bindings. Default value: `/bindings`                                                                                       |
+| `on_install`                                                 | [Call]({{<ref "call">}})    | The call invoked when the App is installed.                                                                                                             |
+| `on_uninstall`                                               | [Call]({{<ref "call">}})    | The call invoked when the App is uninstalled, before the App is removed.                                                                                |
+| `on_version_changed`                                         | [Call]({{<ref "call">}})    | The call invoked when the App needs to be upgraded or downgraded.                                                                                       |
+| `get_oauth2_connect_url`                                     | [Call]({{<ref "call">}})    | The call invoked with the App's "connect to third party" link is selected, to be redirected to the OAuth flow.                                          |
+| `on_oauth2_complete`                                         | [Call]({{<ref "call">}})    | The call invoked when the OAuth flow has successfully completed.                                                                                        |
+| `on_remote_webhook`                                          | [Call]({{<ref "call">}})    | The call invoked when an HTTP webhook is received from a remote system.                                                                                 |
+| `remote_webhook_auth_type`                                   | string                      | Specifies how incoming webhook messages from remote systems should be authenticated by Mattermost.                                                      |
+| `aws_lambda`{{<compass-icon icon-star "Mandatory Value">}}   | [AWSLambda](#aws-lambda)    | Metadata for an App that can be deployed to AWS Lambda and S3 services, and is accessed using the AWS APIs.                                             |
+| `open_faas`{{<compass-icon icon-star "Mandatory Value">}}    | [OpenFAAS](#openfaas)       | Metadata for an App that can be deployed to OpenFAAS.                                                                                                   |
+| `http`{{<compass-icon icon-star "Mandatory Value">}}         | [HTTP](#http)               | Metadata for an App that is already deployed externally and is accessed using HTTP.                                                                     |
+
+{{<note "Mandatory values" "icon-star" "Mandatory Value">}}
+- The `app_id` and `homepage_url` values must be specified.
+- One deployment method - `aws_lambda`, `open_faas`, or `http` - must be specified.
```

Comment:
```suggestion
At least one deployment method - `aws_lambda`, `open_faas`, or `http` - must be specified.
```

Also note, `plugin` is another valid deploy type, perhaps document as a separate PR?

Reply:
Sure; I'll add a card to my board so `plugin` gets documented. :slightly_smiling_face: